android-pdf-viewer/src/main/java/com/github/barteksc/pdfviewer/link/CustomOnTapListener.kt

```diff
@@ -28,28 +28,37 @@ import com.github.barteksc.pdfviewer.annotation.core.shapes.checkIfPointIsInside
 import com.github.barteksc.pdfviewer.listener.OnAnnotationPressListener
 import com.github.barteksc.pdfviewer.listener.OnTapListener
 import com.github.barteksc.pdfviewer.util.logInfo
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.runBlocking
 
 class CustomOnTapListener(
     private val pdfView: PDFView,
     private val pdfUri: Uri,
     private val listener: OnAnnotationPressListener
 ) : OnTapListener {
 
-    override fun onTap(event: MotionEvent): Boolean {
+    override fun onTap(e: MotionEvent?): Boolean {
```

Comment:
Why did this became `null`?

Reply:
This was how it was converted but I was able to remove it